src/test/java/org/apache/commons/lang3/reflect/testbed/ABG2.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.lang3.reflect.testbed;
+
+import org.apache.commons.lang3.ClassUtilsTest;
+
+/**
+ * Test class for {@link ClassUtilsTest#test_isOverridable()}}.
+ */
+public class ABG2 {
```

Comment:
Looking at the rest of the testbed package, this does not have a very descriptive name. I would suggest `ClassWithPackagePrivateMethod` and the method to be `packagePrivateMethod`. The method is not a default method (which applies to interfaces). The purpose of the testbed class is then self-documenting.

The test class that extends this can then be named whatever you want but I would suggest `IsOverridableClass`.